src/support/serde.rs

```diff
@@ -202,32 +251,7 @@ mod tests {
             serialized.push('0');
             serialized.push('"');
             let deserialized = serde_json::from_str::<Uint<BITS, LIMBS>>(&serialized);
-            assert!(deserialized.is_err());
-        });
-    }
-
-    #[test]
-    fn test_serde_zero_invalid_size_error() {
```

Comment:
can you comment here why this test is no longer necessary?

Reply:
Size was checked against pre-calculated length in hex, which is not possible to do anymore because we accept other formats as well